edb/server/connpool/pool.py

```diff
@@ -301,6 +301,8 @@ class BasePool(typing.Generic[C]):
     _blocks: collections.OrderedDict[str, Block[C]]
 
     _is_starving: bool
+    # Indicates if any block is starving for connections, this usually means
+    # the number of active blocks is greater than the pool max capacity.
```

Comment:
Since this is the only comment added in this commit, I find the " pool: Add comments " message a bit misleading :)

Reply:
Oh right - I do intended to add more here, but don't want to delay the release.